OpenRA.Mods.Common/Traits/Render/WithBuildingPlacedOverlay.cs

```diff
@@ -35,6 +35,7 @@ public class WithBuildingPlacedOverlay : INotifyBuildComplete, INotifySold, INot
 	{
 		readonly Animation overlay;
 		bool buildComplete;
+		bool hide;
```

Comment:
Might be a bit more consistent with other code to call this `visible` and invert the checks.